src/biogeophys/WaterStateType.F90

```diff
@@ -268,6 +283,15 @@ subroutine InitHistory(this, bounds, use_aquifer_layer)
             ptr_col=this%wa_col, l2g_scale_type='veg')
     end if
 
+      ! Add excess ice fields to history
+
+      if (use_excess_ice) then
+        data2dptr => this%excess_ice_col(begc:endc,1:nlevsoi)
+        call hist_addfld2d (fname='EXCESS_ICE',  units='kg/m2', type2d='levsoi', &
+             avgflag='A', long_name='excess soil ice (vegetated landunits only)', &
+             ptr_col=this%excess_ice_col, l2g_scale_type='veg')
+       end if
```

Comment:
Looks like an indentation problem at the "end if"

Reply:
Fixed
As well as the indents in the InitCold and Restart

Reply:
Awesome. Looking at the diff to master, I now only see one small indentation issue:

``` fortran
+            g = col%gridcell(c)
+            l = col%landunit(c)
+           if (.not. lun%lakpoi(l)) then  !not lake
+              if (lun%itype(l) == istsoil .or. lun%itype(l) == istcrop) then
```

Reply:
OK, this is fixed.